File: client/src/models/book.ts
export interface Book {
  id: number;
  title: string;
  author: string;
  year: number;
  email: string;
  borrowedByMemberId?: number | null;
  borrowedByMemberName?: string | null;
  isBorrowed: boolean; // Add this field from BookDto
}


export interface NewBook {
  title: string;
  author: string;
  email: string;
  year: number;
}

File: API/Models/Book.cs
namespace api.Models
{
public class Book
{
    public int Id { get; set; }

    public string Title { get; set; } = string.Empty;

    public string Author { get; set; } = string.Empty;

    public string Email { get; set; } = string.Empty;

    public int Year { get; set; }

    public int? BorrowedByMemberId { get; set; }

    public Member? BorrowedByMember { get; set; }
}

}
